internal/store/storageclass.go

```diff
@@ -28,15 +29,27 @@ var (
 	descStorageClassLabelsName          = "kube_storageclass_labels"
 	descStorageClassLabelsHelp          = "Kubernetes labels converted to Prometheus labels."
 	descStorageClassLabelsDefaultLabels = []string{"storageclass"}
+	defaultReclaimPolicy                = v1.PersistentVolumeReclaimDelete
+	defaultVolumeBindingMode            = storagev1.VolumeBindingImmediate
 
 	storageClassMetricFamilies = []metric.FamilyGenerator{
 		{
 			Name: "kube_storageclass_info",
 			Type: metric.Gauge,
 			Help: "Information about storageclass.",
 			GenerateFunc: wrapStorageClassFunc(func(s *storagev1.StorageClass) *metric.Family {
+
+				// Add default values if missing.
+				if s.ReclaimPolicy == nil {
+					s.ReclaimPolicy = &defaultReclaimPolicy
+				}
+
+				if s.VolumeBindingMode == nil {
+					s.VolumeBindingMode = &defaultVolumeBindingMode
+				}
+
 				m := metric.Metric{
-					LabelKeys:   []string{"provisioner", "reclaimPolicy", "volumeBindingMode"},
+					LabelKeys:   []string{"provisioner", "reclaim_policy", "volume_binding_mode"},
```

Comment:
We shouldn't be changing Labelkeys. This becomes a breaking change. Can we revert this please?

Reply:
I changed it because `make e2e` failed locally for me. `promtool` suggested that label names should be in snake case.

```
--- cut ---
check metrics format with promtool
kube_storageclass_info label names should be written in 'snake_case' not 'camelCase'
kube_storageclass_info label names should be written in 'snake_case' not 'camelCase'
--- cut ---
make: *** [e2e] Error 3
```

Reply:
Looks like this new rule was added in the more recent versions of prometheus. This is something we  definitely have to look into. Can you revert this change for now? It will pass in the travis e2e since we use an older version of prometheus that doesn't include this check.